src/StackExchange.Redis/RedisTransaction.cs

```diff
@@ -56,6 +58,12 @@ public Task<bool> ExecuteAsync(CommandFlags flags)
             return base.ExecuteAsync(msg, proc); // need base to avoid our local wrapping override
         }
 
+        internal bool ExecuteInternal(CommandFlags flags, ServerEndPoint endpoint = null)
```

Comment:
Not a huge fan of this, but I also didn't want to change the public API of this for the sake of making a functional test work.